lxd/instance_post.go

```diff
@@ -86,7 +91,19 @@ func instancePost(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(fmt.Errorf("Invalid instance name"))
 	}
 
-	targetNode := queryParam(r, "target")
+	target := queryParam(r, "target")
+	var targetNode, targetGroup string
```

Comment:
Please split this into 2 separate lines as we don't use a single lines for multiple declarations.